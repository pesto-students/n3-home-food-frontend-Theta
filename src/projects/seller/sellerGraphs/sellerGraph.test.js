import * as React from "react";
import Enzyme, { shallow } from "enzyme";
import SellerGraphs from "./sellerGraphs";
import EnzymeAdapter from "enzyme-adapter-react-16";
Enzyme.configure({ adapter: new EnzymeAdapter() });

describe("testing seller graphs component", () => {
  let wrapper;
  test("render without error", () => {
    wrapper = shallow(<SellerGraphs />);
    expect(wrapper.length).toBe(1);
  });

  describe("Seller Graph Component", () => {
    it('should render correctly in "debug" mode', () => {
      const component = shallow(<SellerGraphs />);

      expect(component).toMatchSnapshot();
    });
  });
});
