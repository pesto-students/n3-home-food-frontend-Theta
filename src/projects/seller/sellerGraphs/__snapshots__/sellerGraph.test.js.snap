// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing seller graphs component MyComponent should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;

exports[`testing seller graphs component Seller Graph Component should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;
